src/encoded/types/dataset.py

```diff
@@ -865,7 +865,45 @@ class Series(Dataset, CalculatedSeriesAssay, CalculatedSeriesAssayType, Calculat
                 'pipeline_version',
                 'title',
             ],
-        )
+        ),
+        Path('series_files', include=[
```

Comment:
Formatting nit: these arguments could each have newline to make indentation clearer.